api/interservice/license_control/license-control.proto

```diff
@@ -4,7 +4,7 @@ package chef.automate.domain.license_control;
 option go_package = "github.com/chef/automate/api/interservice/license_control";
 
 import "google/protobuf/timestamp.proto";
-import "github.com/chef/automate/lib/license/license.proto";
+import "lib/license/license.proto";
```

Comment:
This change fixes reflection

Reply:
We've got a test helper for that somewhere; having run into the same bit before. 
👉 https://github.com/chef/automate/blob/master/lib/grpc/grpctest/reflection.go#L16